modules/ROOT/pages/dw-values-functions-update.adoc

```diff
@@ -0,0 +1,287 @@
+= update
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+
+[[update1]]
+== update&#40;Object, String&#41;: Any
+
+This `update` function updates a field in an object with the specified
+string value.
+
+
+The function returns a new object with the specified field and value.
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `objectValue` | The object to update.
+| `fieldName` | A string that provides the name of the field.
+|===
+
+=== Example
+
+This example updates the `name` field in the object `{name: "Mariano"}` with
+the specified value.
+
+==== Source
+
+[source,DataWeave,linenums]
+----
+%dw 2.0
+import * from dw::util::Values
+output application/json
+---
+{name: "Mariano"} update "name" with "Data Weave"
+----
+
+==== Output
+
+[source,Json,linenums]
+----
+{
+  "name": "Data Weave"
+}
+----
+
+
+[[update2]]
+== update&#40;Object, PathElement&#41;: Any
+
+This `update` function updates an object field with the specified
+ `PathElement` value.
+
+
+The function returns a new object with the specified field and value.
+
+=== Parameters
+
+[%header, cols="1,3"]
+|===
+| Name   | Description
+| `objectValue` | The object to update.
+| `fieldName` | A `PathElement` that specifies the field name.
+|===
+
+=== Example
+
+This example updates the value of a `name` field in the object
+`{name: "Mariano"}`. It uses `field("name")` to return the `PathElement`
+that it passes to `update`. It uses `with "Data Weave"` to specify the value
+(`Data Weave`) of `name`.
+
+==== Source
+
+[source,DataWeave,linenums]
+----
+%dw 2.0
+import * from dw::util::Values
+output application/json
+---
+{name: "Mariano"} update field("name") with "Data Weave"
+----
+
+==== Output
+
+[source,Json,linenums]
+----
+{
+  "name": "Data Weave"
+}
+----
+
+
+[[update3]]
+== update&#40;Array, Number&#41;: Any
+
+Updates an array index with the specified value.
+
+This `update` function returns a new array with that changes the value of
```

Comment:
remove **with**

Reply:
done in adoc file only, will update dw repo separately

Reply:
done in dw repo